python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -672,6 +722,169 @@ def load_context(self, params: PipesParams) -> Iterator[PipesContextData]:
             )
 
 
+class PipesOutErrLogWriter(
```

Comment:
I think `PipesStdioLogWriter` is a better name since "stdio" is a conventional abbreviation